_posts/2019-09-10-plipnick-Living_the_Dream_with_Continuous_Improvements.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+author: plipnick 
+title: Living the Dream with Continuous Improvements
+---
+
+
+<img src="https://technologyandoptimizingprocesses.files.wordpress.com/2019/08/morocco-20-copy-1.jpg"
```

Comment:
Could you add a summary teaser followed by <!--more--> on a new line? This will make the blog index render cleanly.
You can see an example here: https://raw.githubusercontent.com/devopsdojoconsortium/devopsdojoconsortium.github.io/release/_posts/2019-09-07-bfinster.md